addon/components/u-edit/shared-triggers/modals/file-uploader.hbs

```diff
@@ -27,7 +27,7 @@
 
           {{file-uploader
             file=this.droppedFile allowedExtensions=this.allowedExtensions headers=this.uploaderHeaders
-            extra=this.uploaderExtra text=(t (concat "uedit_editor.toolbar." this.titleKey ".browse"))
+            extra=this.uploaderExtra maxSize="10 MB" text=(t (concat "uedit_editor.toolbar." this.titleKey ".browse"))
```

Comment:
does it correctly handle the error if > 10 Mb ? if not, it's my bad not doing it :(

Reply:
You're sure the format is defined in MB with a space ?

Reply:
Yes! It displays the correct error

Reply:
@Miexil Yep